profiles/johnmiicheal.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://i.imgur.com/9lY2CQQ.png",
```

Comment:
```suggestion
  "image": "https://camo.githubusercontent.com/5ed97aab374dec9381e88b21a74c19653a22b47fd92f57fa6259b8205edee1ab/68747470733a2f2f692e696d6775722e636f6d2f396c59324351512e706e67",
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix